src/DynamoCore/Graph/Nodes/CodeBlockNode.cs

```diff
@@ -658,6 +658,21 @@ internal void ProcessCodeDirect(ProcessCodeDelegate handler, bool isCodeBlockNod
             OnNodeModified();
         }
 
+        internal void Warning(string p)
+        {
+            State = ElementState.MigratedFormula;
+            if (!Infos.Any(x => x.Message.Equals(p)))
+            {
+                var texts = p.Split(new[] { System.Environment.NewLine }, StringSplitOptions.None);
+                var infoList = new List<Info>();
+                foreach (var text in texts)
+                {
+                    infoList.Add(new Info(text, State));
+                }
+                Infos.AddRange(infoList);
+            }
+        }
+
```

Comment:
Can you add a description to this method ? Looks like it is specific to Migrated formulas